workspace/__ardulib__/Graphics/Bitmap.h

```diff
@@ -0,0 +1,25 @@
+
+#ifndef BITMAP_H
+#define BITMAP_H
+
+/*
+ * A structure to hold basic bitmap parameters.
+ */
+struct Bitmap {
+    Bitmap(uint8_t* newBuffer = nullptr, uint8_t newColorDepth = 0, uint16_t newWidth = 0, uint16_t newHeight = 0, uint16_t newAlphaColor = 0x0000) {
+        buffer = newBuffer;
+        colorDepth = newColorDepth;
+        width = newWidth;
+        height = newHeight;
+        alphaColor = newAlphaColor;
```

Comment:
I still thinking that alpha color should be defined in another layer of abstraction 😈
BTW, it will save an extra 2 bytes for each bitmap, which has no transparency at all and could make a more extensible and flexible system 😛